spec/services/admin/flags/dismiss_spec.rb

```diff
@@ -1,7 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe Admin::Flags::Dismiss do
-  subject(:service) { described_class.call(flag: flag) }
+  let(:admin) { create(:user, admin: true) }
```

Comment:
Can we add a test for setting the resolved by id in this service? We should hopefully be able to follow the same pattern as the action_taken and status tests in here.